R/tm_outliers.R

```diff
@@ -115,6 +139,27 @@
 #'   shinyApp(app$ui, app$server)
 #' }
 #'
+#' # Decorators
+#' function_decorator <- function(p) {
+#'   p <- p + ggplot2::ggtitle("A new title")
+#' }
+#'
+#' quote_decorator <- quote({
+#'   g <- g + ggplot2::ggtitle("A new title")
+#' })
+#'
+#' module_decorator <- teal_transform_module(
+#'   ui = function(id) NULL,
+#'   srv = function(id, data) {
+#'     within(
+#'       data,
+#'       {
+#'         g <- g + ggplot2::ggtitle("A new title")
```

Comment:
We could unify, so it's always `graph` or `plot` in this module.

Reply:
That would be nice! `plot` sounds great to me and it's the most used in modules, but as you said in the meeting the "G" in `TLGs` is for graph :laughing:

Reply:
Let's go with plot for now